Selenium.HtmlElements/src/Proxy/WebElementProxy.cs

```diff
@@ -29,7 +29,20 @@ protected override void Execute(Action<IWebElement> action)
 
                     continue;
                 }
+                catch (InvalidOperationException ex)
+                {
+                    if (ex.Message.Contains("Element does not exist in cache (status: 10)"))
+                    {
+                        Loader.Reset();

```

Comment:
Same operation is done twice. Please use single catch clause to avoid code duplication. Could look like following:
```C#
catch (Exception ex){
    if(!IsStaleElementReferenceException(ex)){
        throw;
    }

    // handling
}
```